RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -169,4 +280,94 @@ public protocol PaywallViewControllerDelegate: AnyObject {
     optional func paywallViewControllerDidAppear(_ controller: PaywallViewController)
 }
 
+// MARK: - Private
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, *)
+private extension PaywallViewController {
+    func createHostingController() -> UIHostingController<PaywallContainerView> {
+        let purchaseHandler = PurchaseHandler()
+        purchaseHandler.refreshSubscriptions = refreshSubscriptions // ASPETTA GLI INIT
```

Comment:
this looks unused?
it's supposed to end up in `configuration`

Reply:
yes, moved thx!